### ISSUE TITLE
cannot extract audio file name

### ISSUE THREAD

**Issue by adubkov** [yinyanghu/cuefix#2]
I'm getting this `cannot extract audio file name` because regex explicitly looking for patter in upper case characters: https://github.com/yinyanghu/cuefix/blob/e72272baaeded6256ee88b196402e2fc042ad403/cuefix/cuefix.py#L17

But in my case it was:
```
FILE "..." wave
```

**Comment by yinyanghu** [comment]
Yes, you are right. I didn't consider this case. Feel free to make a pull request and I could review and approve it. Thanks.

**Comment by seascape** [comment]
Just ran into this myself on first-ever run. I don't mean this as an attack, but the app doesn't seem very robust if it can't handle slight, normal variations in filenames.
